apps/client/src/routes.js

```diff
@@ -90,3 +90,8 @@ export const redirectConfig = [
 export const routes = Object.fromEntries(
   routeConfig.map(({ name, path }) => [name, path])
 );
+
+export const autofillRoutes = {
+  address: routes.location,
+  // map: ...
+};
```

Comment:
Do we need this in this PR?

Reply:
Already gone. Forked from the wrong branch, sorry.
I will squash it before merging.